### PR TITLE
cosmoXplore : APoD using Vanilla JS

### PR DESCRIPTION
**Pull request by PranavBarthwal**
# Pull Request for WebMasterLog 💡

## Issue Title: cosmoXplore : APoD using Vanilla JS

- **Name of the project** :  cosmoXplore : APoD
- **Category:**  Vanilla JS
- **Name:**  Pranav Barthwal
- **GitHub ID:**  https://github.com/PranavBarthwal
- **Idenitfy yourself:**  Contributor under JWoC '24

Closes: #73 

### Describe the add-ons or changes you've made 📃
I've added APoD project keeping in mind the contributor guidelines

Give a clear description of what have you added or modifications made
JS App that fetches Astronomy Picture of The Day using NASA APoD API

## Type of change ☑️

What sort of change have you made:

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested? ⚙️

Describe how it has been tested 
Describe how have you verified the changes made
I have tested the program by running it on my local system by cghecking whether APoD is displayed or not,

## Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots 📷

Screenshot 2024-02-06 004242

### CODE REVIEW

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

File: Vanilla-JS-Projects/README.md
[the remark this answers is not in the document]
Comment (PranavBarthwal) [reply Vanilla-JS-Projects/README.md:1]: done

File: Vanilla-JS-Projects/README.md [hunk 1]
```diff
@@ -56,8 +56,7 @@ To get started with these Vanilla JavaScript projects, follow these steps:
 |   9   | [Hex Color Code Generator](./Intermediate/Hex-color-code-generator-website) | ![Intermediate](https://img.shields.io/badge/Intermediate-FFD700?style=for-the-badge) |
 |  10   | [Countdown Timer](./Intermediate/Countdown-Timer) | ![Intermediate](https://img.shields.io/badge/Intermediate-FFD700?style=for-the-badge) |
 |  11   | [Recipe Finder](./Advanced/Recipe-Finder/) | ![Advanced](https://img.shields.io/badge/Advanced-FF0000?style=for-the-badge) |
-|  10   | [Morse Code Generator](./Intermediate/Morse-Code-Generator) | ![Intermediate](https://img.shields.io/badge/Intermediate-FFD700?style=for-the-badge) |
```
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/README.md:59]: Don't remove the Morse Code Generator, enlist it and sequence the numbering.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Comment by PranavBarthwal** [comment]
@Avdhesh-Varshney done.

**Review by Avdhesh-Varshney — APPROVED** [review]
PR Approved 🎉

**Pull request merged**